src/App.js

```diff
@@ -89,13 +90,22 @@ class App extends React.Component {
               </NavDropdown>
             </Nav>
             <Form inline>
-              <FormControl type="text" placeholder="Search" className="mr-sm-2" />
+              <FormControl value={this.state.searchText}  onChange={(e) => this.handleSearchChange(e)} type="text" placeholder="Search" className="mr-sm-2" />
```

Comment:
If we made it an anonymous function on line 100, do we need to it again on this onChange?

Reply:
That's a great question, Loi! I think we need to do it here, because this is a separate function than the one on line 100.